press/api/marketplace.py

```diff
@@ -317,7 +384,7 @@ def create_site_for_app(
 	subdomain: str,
 	apps: list[dict],
 	cluster: str,
-	site_plan: str,
+	site_plan: str = "$10",
```

Comment:
why?

Reply:
Ah good catch, all the changes here were things I was trying out. Reverting them all.